features/serializer/immutbale_attributes_through_serialization_groups.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Make attributes of an entity immutable with serialization groups
+  In order to have a resource with an immutable attribute
+  As a client
+  I can set the attribute while creating a resource, but cannot update it later
+
+  @createSchema
+  Scenario: Set an immutable attribute while creating a resource and try to change it
+    When I add "Content-Type" header equal to "application/ld+json"
+    And I send a "POST" request to "/dummy_entity_with_immutables" with body:
+    """
+    {
+      "immutableName": "foo",
+      "mutableWebsite": "katzensaft.de"
+    }
+    """
+    Then the response status code should be 201
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
+    And the JSON should be equal to:
+    """
+    {
+      "@context": "/contexts/DummyEntityWithImmutable",
+      "@id": "/dummy_entity_with_immutables/foo",
+      "@type": "DummyEntityWithImmutable",
+      "immutableName": "foo",
+      "mutableWebsite": "katzensaft.de"
+    }
+    """
+    Then I add "Content-Type" header equal to "application/ld+json"
+    And I send a "PUT" request to "/dummy_entity_with_immutables/foo" with body:
+    """
+    {
+      "immutableName": "bar",
+      "mutableWebsite": "katzensaft.de"
+    }
+    """
+    Then the response status code should be 200
```

Comment:
Shouldn't it result in 404 instead?

Reply:
Maybe, but this is the current behavior of the api platform. Attributes that are read-only because of serialization groups are just dropped, see this test for example:
https://github.com/api-platform/core/blob/2f941fc3c0dc9dd257da3b34a6bb3f5c21c4230c/features/serializer/group_filter.feature#L717

Reply:
Anyway, changing the response code would be way out of the scope of this mr